proxyd/server.go

```diff
@@ -347,6 +378,16 @@ func (s *Server) handleBatchRPC(ctx context.Context, reqs []json.RawMessage, isB
 			continue
 		}
 
+		// Take rate limit for specific methods.
+		// NOTE: eventually, this should apply to all batch requests. However,
+		// since we don't have data right now on the size of each batch, we
+		// only apply this to the methods that have an additional rate limit.
```

Comment:
Should it matter if rate limiting isn't applied to batch requests that don't contain method overrides? I could be wrong, but isn't the global rate limiter meant only for downstream requests? Which we already do in `handleRPC`.